### PR TITLE
Task/sdk 3803/callbacks not working with workmanager

### PR DESCRIPTION
**Pull request by Anush-Shand**
[Flutter Callback Dev Test Sheet](https://docs.google.com/spreadsheets/d/1aOYf7jd5AuO7gdKP7JlatfZZTIm1pUQbBCr1mOXohVs/edit#gid=0)

https://wizrocket.atlassian.net/browse/SDK-3803

### CODE REVIEW

File: android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java [hunk 1]
```diff
@@ -93,13 +95,14 @@ public class CleverTapPlugin implements ActivityAware,
 
     private MethodChannel dartToNativeMethodChannel;
 
-    private static MethodChannel nativeToDartMethodChannel;
+    private MethodChannel nativeToDartMethodChannel;
```
Comment (CTLalit) [inline android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java:98]: rename to `lastNativeToDartMethodChannel`

**Comment by shivamsharma2710** [comment]
@Anush-Shand Please upgrade the SDK version in `pubSpec` and increment the **libVersion** in dart file.

File: android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java [hunk 2]
```diff
@@ -93,13 +95,14 @@ public class CleverTapPlugin implements ActivityAware,
...
 
-    private static MethodChannel nativeToDartMethodChannel;
+    private MethodChannel lastNativeToDartMethodChannel;
 
     private CleverTapAPI cleverTapAPI;
 
     private Context context;
 
     public static Map<String, Object> variables = new HashMap<>();
+    public static Set<MethodChannel> nativeToDartMethodChannelSet = new HashSet<>();
```
Comment (CTLalit) [inline android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java:105]: make private

**Review by CTLalit — APPROVED** [review]
Approved with small comments.

**Pull request merged**